plaidml/bridge/openvino/src/plaidml_plugin/ops/pad.cpp

```diff
@@ -13,27 +13,36 @@
 using namespace plaidml;          // NOLINT[build/namespaces]
 using namespace InferenceEngine;  // NOLINT[build/namespaces]
 
+namespace {
+
+template <typename T>
+std::vector<T> cast_constant_operand(size_t operand_idx, ngraph::Node* layer) {
+  auto ngraph_const =
+      std::dynamic_pointer_cast<ngraph::op::Constant>(layer->input_value(operand_idx).get_node_shared_ptr());
+  if (ngraph_const) {
+    return ngraph_const->cast_vector<T>();
+  } else {
+    THROW_IE_EXCEPTION << "Dynamic slicing not currently supported by PlaidML plugin; all of begin, end, and stride "
+                          "must be Constants.";
```

Comment:
This exception text was specific for StridedSlice, update it for Pad (e.g. Dynamic padding not currently supported by PlaidML plugin; all of pads_begin, pads_end, and pads_value must be Constants).

(We could also create a shared cast_constant_operand in plaidml_utils with some sort of joint error message; IMO the current approach is sufficient.)